ivy/functional/frontends/jax/lax/linalg.py

```diff
@@ -38,3 +38,25 @@ def svd(x, /, *, full_matrices=True, compute_uv=True):
     if not compute_uv:
         return ivy.svdvals(x)
     return ivy.svd(x, full_matrices=full_matrices)
+
+
+@to_ivy_arrays_and_back
+def triangular_solve(
+    a,
+    b,
+    *,
+    left_side=False,
+    lower=False,
+    transpose_a=False,
+    conjugate_a=False,
+    unit_diagonal=False,
+):
+    return ivy.solve(
```

Comment:
[`ivy.solve`](https://unify.ai/docs/ivy/docs/functional/ivy/linear_algebra/ivy.functional.ivy.linear_algebra.solve.html#solve) does not take these as parameters. Maybe you could take a look of other PRs at the [todo list](https://github.com/unifyai/ivy/issues/1582) such as [this](https://github.com/unifyai/ivy/pull/4585/files) to see how other contributors work on situations like this. Thanks